Cargo.toml

```diff
@@ -8,15 +8,15 @@ members = [
 ]
 
 [workspace.package]
-version = "0.4.12"
+version = "0.5.0"
```

Comment:
We'll be versioning this shortly as part of #224, so this may need to back out if that beats this into the repo (but you can leave for now).